### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Anubhav-developr**
I had done some minor changes.

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

**Comment by luzpaz** [comment]
Please close this ticket and refer changes to #3998. Thx

**Comment by luzpaz** [comment]
@Anubhav-developr :point_up:

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ pages below. Once this is done, FreeCAD can be simply compiled with
 cMake. On Windows, these libraries are bundled and offered by the 
 FreeCAD team in a convenient package. On Linux, they are usually found 
 in your distribution's repositories, and on Mac OSX and other platforms 
-you will usually need to compile them yourself.
+you will usually need to compile them by yourself.
```
Comment (luzpaz) [inline README.md:81]: This is superfluous, IMHO.

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ your model history and changing its parameters.
  Whether you are a hobbyist, a programmer, an experienced CAD user, 
  a student or a teacher, you will feel right at home with FreeCAD.
 
-* **Cross platform** FreeCAD runs on Windows, Mac and Linux
+* **Cross platform** FreeCAD runs on Windows, Mac and Linux Operating systems.
```
Comment (luzpaz) [inline README.md:40]: Superfluous. Also 'Operating' shouldn't be capitalized.

**Review by luzpaz — COMMENTED** [review]
I don't think these changes are necessary. Recommend closing issue.

**Comment by wwmayer** [comment]
Following luzpaz' advice and close this PR.

**Pull request closed without merging**